Xamarin.Forms.Platform.Tizen/Renderers/TabbedPageRenderer.cs

```diff
@@ -403,5 +432,126 @@ void OnCurrentPageChanged()
 			Element.UpdateFocusTreePolicy();
 			_isUpdateByCurrentPage = false;
 		}
+
+		void ApplyBarItemColors(EToolbarItem item, BarItemColorType type, EColor color)
```

Comment:
I think the `color` parameter here is unnecessary. It can always be obtained based on the `BarItemColorType` value.

Reply:
Opinion makes sense, but I think it kinds of a matter of choice (not a big deal). Removing parameters will require additional conditions in this method.